lua/orgmode/org/mappings.lua

```diff
@@ -258,6 +258,58 @@ function OrgMappings:change_date()
   Calendar.new({ callback = cb, date = date }).open()
 end
 
+function OrgMappings:set_priority()
+  local item = Files.get_closest_headline()
+  local todo = item.todo_keyword
+  local high_priority = config.org_priority_highest
+  local low_priority = config.org_priority_lowest
+  local is_alphabetical = string.find(high_priority, '%a')
+
+  local start = high_priority
+  local fin = low_priority
+  if is_alphabetical then
+    start = string.byte(high_priority)
+    fin = string.byte(low_priority)
+  end
+
+  local enumerated = {}
+  for i = start, fin do
```

Comment:
Priority count could be high, probably good to bound this somewhere.

Reply:
Actually emacs allows 3 level of priorities, i don't think it can go beyond that. This looks fine to me.

Reply:
Pulling it from the quote:

>When using numeric priorities, you need to set org-priority-highest, org-priority-lowest and org-priority-default to integers, which must all be strictly inferior to 65

mentioned [here](https://orgmode.org/manual/Priorities.html).